.changes/v2.17.0/512-improvements.md

```diff
@@ -0,0 +1,4 @@
+* Added VCDClient.GetAllNsxtEdgeClusters for lookup of NSX-T Edge Clusters in wider scopes -
+  Provider VDC, VDC Group or VDC [GH-512]
+* Switch VDC.GetAllNsxtEdgeClusters to use 'orgVdcId' filter instead of '_context' (now deprecated)
```

Comment:
An ask for the PR description: would be useful to see the initiating issue number or the reason why we are adding this.

Reply:
> An ask for the PR description: would be useful to see the initiating issue number or the reason why we are adding this.

Put a reference to Terraform PR and also short explanation why it is needed.